docs/README.md

```diff
@@ -7,13 +7,22 @@ The docs are built using [Docusaurus 2](https://v2.docusaurus.io/).
 ### Installation
 
 ```
-$ yarn
+$ make install-docs
```

Comment:
Before developer executes this line we need to let them know that they need to install Rasa python dependencies, as specified:
```
State that developer needs to install python dependencies for Rasa with make install
```
in [JIRA ticket](https://rasahq.atlassian.net/browse/ATO-843)